lib/midi-smtp-server.rb

```diff
@@ -223,6 +226,7 @@ def ssl_context
     # +max_connections+:: maximum number of connections, this does limit the number of concurrent TCP connections (not set or nil => unlimited)
     # +crlf_mode+:: CRLF handling support (:CRLF_ENSURE [default], :CRLF_LEAVE, :CRLF_STRICT)
     # +do_dns_reverse_lookup+:: flag if this smtp server should do reverse DNS lookups on incoming connections
+    # +io_sleep_time+:: time in seconds to sleep on IO::WaitReadable exception (DEFAULT_IO_SLEEP_TIME)
```

Comment:
+ io_waitreadable_sleep +:: seconds to sleep in loop when no input data is available (DEFAULT_IO_SLEEP_TIME)